docs/faq.md

```diff
@@ -334,7 +350,8 @@ include = ["*.py", "*.pyi", "**/pyproject.toml", "*.ipynb"]
 ```
 
 This will prompt Ruff to discover Jupyter Notebook (`.ipynb`) files in any specified
-directories, and lint them accordingly.
+directories, and lint them accordingly. Formatter support is not yet available for Jupyter
+Notebooks.
```

Comment:
Do you mind updating this as the PR has been merged?

Also, this is another section where we could recommend `extend-include` instead for including Jupyter Notebook files if we decide to do so.